zokrates_cli/examples/book/assert.zok

```diff
@@ -1,3 +1,3 @@
 def main() -> ():
-	assert(1f == 2f)
+	assert(2f == 2f)
```

Comment:
@Schaeff I'm not sure about this example, I had to change it to a passing one otherwise the tests will fail. Any thoughts?

Reply:
Yes that's okay. Maybe put `1 + 1 == 2`